Src/Web/Web.Shared.Net/RequestTrackingTelemetryModule.cs

```diff
@@ -283,6 +257,10 @@ public void AddTargetHashForResponseHeader(HttpContext context)
                     if (this.correlationIdLookupHelper.TryGetXComponentCorrelationId(requestTelemetry.Context.InstrumentationKey, out correlationId))
                     {
                         context.Response.Headers.SetNameValueHeaderValue(RequestResponseHeaders.RequestContextHeader, RequestResponseHeaders.RequestContextCorrelationTargetKey, correlationId);
+
+                        // set additional header that allows to read this Request-Context from Javascript SDK
+                        // append this header with additional value to the potential ones defined by customer and they will be concatenated on cliend-side
+                        context.Response.AppendHeader(RequestResponseHeaders.AccessControlExposeHeadersHeader, RequestResponseHeaders.RequestContextHeader);
```

Comment:
make a config setting for this so it can be turned off if needed, please. I can see customers want to hide this information from exposure to third party web sites

Reply:
Makes sense. Added EnableAccessControlExposeHeader - are you ok with the name?